Runner.php

```diff
@@ -15,34 +16,28 @@ public function __construct($config, $server)
 
     public function queueDirectory($dir, $prefix = '')
     {
-        if(isset($this->config['include_paths'])) {
-            $this->queueWithIncludePaths();
-        } else {
-            $this->queuePaths($dir, $prefix, $this->config['exclude_paths']);
-        }
-
+        $this->queueWithIncludePaths();
         $this->server->process_work(false);
     }
 
     public function queueWithIncludePaths() {
         foreach ($this->config['include_paths'] as $f) {
             if ($f !== '.' and $f !== '..') {
+                if ($f == "./") {
+                    $f = "";
+                }
 
-                if (is_dir("/code$f")) {
```

Comment:
If you just set and work with (something like) `absolutePath = "/code/$f"` would that solve everything?

Seems more logically consistent: if the SPEC says each `$f` will be a path relative to `/code`, then `/code/$f` should always be an accurate absolute. If you get an `$f` that doesn't work under that system, I'd say it's a bug in the engine-runner, not the engine.